pkg/cli/admin/nodeimage/create.go

```diff
@@ -471,6 +472,10 @@ func (c *BaseNodeImageCommand) fetchClusterReleaseImage(ctx context.Context) (st
 			return "", fmt.Errorf("command expects a connection to an OpenShift 4.x server")
 		}
 	}
+	// Adds a guardrail for node-image commands which is supported only for Openshift version 4.17 and later
+	if isVersionBefore(cv.Status.Desired.Version, 4, 17) {
```

Comment:
Let's use an existing library to check versions. Here's an example: https://github.com/openshift/assisted-installer/blob/37fd43a46b3743208c1ddea98cf3b98faf0792ad/src/common/common.go#L223